core/src/test/scala/org/broadinstitute/dsde/rawls/billing/AzureBillingProjectLifecycleSpec.scala

```diff
@@ -831,30 +841,7 @@ class AzureBillingProjectLifecycleSpec extends AnyFlatSpec {
     }
   }
 
-  behavior of "initiateDelete"
-
-  it should "return None for the jobId if the landing zone does not exist" in {
```

Comment:
`BillingProjectOrchestrator` now gets the `landingZoneId` and only calls `cleanupLandingZone` if one exists.